pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "imgparse"
-version = "1.18.1"
+version = "1.18.2"
```

Comment:
Bump to 1.19.0 since you are adding a new function. Also make sure to update _version.py